### ISSUE TITLE
Cannot find module 'manticoresearch-ts' or it's type declarations

### ISSUE THREAD

**Issue by Lamarcke** [manticoresoftware/manticoresearch-typescript#1]
The following error appears when copying the imports from the `README`:  

![Screenshot_20231016_162155](https://github.com/manticoresoftware/manticoresearch-typescript/assets/23425058/77869b97-7dde-4418-a5d7-cebb58fdb064)

**Fix**: 

```typescript
import * from "manticoresearch-ts/dist/src"

// OR

import * from "manticoresearch-ts/dist/src/index"
```

This probably happens because the `index.d.ts` file is under `dist/src` in the npm package.

**Comment by Nick-S-2018** [comment]
We've fixed Readme as proposed

**Comment by mannol** [comment]
@Nick-S-2018 Fix proposed here is a hack, checkout #2 for proper fix.
